homeassistant/components/telnet/switch.py

```diff
@@ -60,11 +62,11 @@ def setup_platform(
     discovery_info: DiscoveryInfoType | None = None,
 ) -> None:
     """Find and return switches controlled by telnet commands."""
-    devices = config.get(CONF_SWITCHES, {})
+    devices: dict[str, Any] = config.get(CONF_SWITCHES, {})
     switches = []
 
     for object_id, device_config in devices.items():
-        value_template = device_config.get(CONF_VALUE_TEMPLATE)
+        value_template: Template = device_config.get(CONF_VALUE_TEMPLATE)
```

Comment:
The template is optional.
```suggestion
        value_template: Template | None = device_config.get(CONF_VALUE_TEMPLATE)
```